javascript/cable_ready.js

```diff
@@ -297,6 +297,13 @@ const DOMOperations = {
     dispatch(document, 'cable-ready:after-remove-storage-item', operation)
   },
 
+  scrollIntoView: operation => {
+    const { element } = operation
+    dispatch(element, 'cable-ready:before-scroll', operation)
+    if (!operation.cancel) element.scrollIntoView(operation)
+    dispatch(element, 'cable-ready:after-scroll', operation)
+  },
```

Comment:
We should update the dispatched event names to `cable-ready:before-scroll-into-view` and `cable-ready:after-scroll-into-view`.

Reply:
Nice catch!